synapse/lib/stormlib/gen.py

```diff
@@ -42,6 +42,37 @@ class LibGen(s_stormtypes.Lib):
                       {'name': 'cve', 'type': 'str', 'desc': 'The CVE id.'},
                   ),
                   'returns': {'type': 'storm:node', 'desc': 'A risk:vuln node with the given CVE.'}}},
+
+        {'name': 'riskThreat', 'desc': 'Returns risk:vuln node by CVE, adding the node if it does not exist.',
+         'type': {'type': 'function', '_funcname': '_storm_query',
+                  'args': (
+                      {'name': 'name', 'type': 'str', 'desc': 'The reported name of the threat cluster.'},
+                      {'name': 'reporter', 'type': 'str', 'desc': 'The name of the organization which reported the threat cluster.'},
+                  ),
+                  'returns': {'type': 'storm:node', 'desc': 'A risk:threat node.'}}},
+
+        {'name': 'riskToolSoftware', 'desc': 'Returns risk:vuln node by CVE, adding the node if it does not exist.',
```

Comment:
Copypasta. Suggest "Returns risk:tool:software node based on the software/malware and reporter names, adding the node if it does not exist."